### ISSUE TITLE
Better `tfds.as_dataframe` visualization

### ISSUE THREAD

**Issue by Conchylicultor** [tensorflow/datasets#2791]
TFDS visualisation is helpful but currently a little too limited:

For example, [COCO](https://www.tensorflow.org/datasets/catalog/coco) visualisation makes it difficult to view the bounding boxes, labels:

![image](https://user-images.githubusercontent.com/9047355/100262249-2df54200-2f4c-11eb-9786-3ba9d90b96f6.png)

Here are a few ideas to improve the current visualisation:
Done:

* [x] `BBoxFeature` inside `Sequence`: Display some boxes in some empty 150x150px image
* [x] Video could also be greatly improved. If fmmpeg is found, it would be nice to generate a proper `.mp4` video which would be more compressed than `.gif` and would have play/pause controls.
* [x] The video framerate should also be greatly reduced. Currently, video like https://www.tensorflow.org/datasets/catalog/abstract_reasoning are much too fast

Todo:

* [x] `ClassLabel`/`Tensor` inside `Sequence`: Display individual `repr_html`, once per line in a `div` with scrollbars

For the implementation, dataframe will automatically call the `. repr_html()` or `repr_html_batch` of the corresponding feature.

For example:

```python
features={
    'image': tfds.features.Image(),
    'objects': tfds.features.Sequence({
        'bbox': tfds.features.BBoxFeature(),
    })
}
```
Will call `tfds.features.Image.repr_html` and `tfds.features.BBoxFeature.repr_html_batch`.

https://github.com/tensorflow/datasets/blob/ad5373e97c37569e9bf024840ba6d6503f8b1116/tensorflow_datasets/core/features/audio_feature.py#L89

**Comment by harsh020** [comment]
@Conchylicultor Hi, I was working on it. Could you please tell me if there is any way to preview **locally** how my changes will be rendered in the frontend?

Thank You.

**Comment by Conchylicultor** [comment]
> Could you please tell me if there is any way to preview locally how my changes will be rendered in the frontend?

Thank you @harsh020 . To preview locally, you can either:
* Test locally in a local jupyter notebook.
* Save the `tfds.as_dataframe(...)._repr_html_()` content in a `.html` file and view it on your browser.

> The implementations will be in repr_html_batch, isn't it?

Yes, for all features wrapped inside `Sequence` (e.g. `Sequence((my_feature))`), `my_feature.repr_html_batch(ex)` will be called during visualization. For other features, `repr_html` is called.

**Comment by harsh020** [comment]
@Conchylicultor Hi, the last task of grouping `ClassLabels` remain. Will open a PR ASAP.

Thank You.

**Comment by harsh020** [comment]
Fixed by PRs `#2798` and `#2887`
